website/content/blog/reducing-your-environmental-impact-with-the-linkerd-service-mesh.md

```diff
@@ -0,0 +1,54 @@
+
+##  Additional software = bigger resource footprint? Not necessarily.   
+
+Each additional piece of software translates into consuming more system resources, right? In theory, yes, but if it helps you run your system more efficiently, you may actually decrease your footprint. 
+
+The environmental impact of adding features such as zero-config and in-transit encryption (mTLS), as provided by Linkerd, is not zero — no software is. Whether you think that environmental (and financial) cost is viable or reasonable is a different discussion. But you might be surprised to hear that Linkerd can minimize your impact even when compared to no service mesh. That's thanks to Linkerd's intelligent load balancing. But let's take a step back and examine how Kubernetes works first.
+
+##  Distributing work improves with Linkerd over Native Kubernetes
+
+Kubernetes allows us to more densely pack processes on computers. Servers have a certain capacity to do work, but there is always a gap between how much work they _can_ do and how much the system asks them to do. Kubernetes schedules work more efficiently on existing servers (bin packing), allowing us to run more stuff on the same hardware — a big win! 
+
+Linkerd improves those efficiency gains even further by taking a different approach to endpoint selection. While Kubernetes balances its load based on connections, Linkerd instead does it based on requests. As a result, Linkerd saves you significant compute resources. Let's break that down.  
+
+In the old world, applications would make one connection, send a request, get an answer, and close that connection — not very efficient when you have hundreds or thousands of requests per minute. Today, HTTP/2 and gRPC make a connection once and send multiple requests to avoid this inefficiency.
+
+Kubernetes doesn't handle that out-of-the-box, though. Natively, Kubernetes simply directs the entire connection to a Pod and doesn't worry about how many requests are coming down the connection. This can be problematic: if you're running 10 Pods for a particular workload, and 50 requests come down a single connection, Kubernetes will send all 50 to the same single Pod. In the best case, the other nine Pods just sit idle; in the worst case, the one Pod doing all the work wasn't sized for this peak load, resulting in a crash.
```

Comment:
```suggestion
Kubernetes doesn't handle that out-of-the-box, though. Natively, Kubernetes simply directs the entire connection to a Pod and doesn't worry about how many requests are coming down the connection. This can be problematic: if you're running 10 Pods for a particular workload, and 50 requests come down a single connection, Kubernetes will send all 50 to the same single Pod. In the best case, the other nine Pods just sit idle; in the worst case, the one Pod doing all the work wasn't sized for this peak load and crashes.
```